.projen/tasks.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
Many of these scripts were automatically created by Projen, I may have the ability to override them if some should be removed

Reply:
Yeah, I'm not sure all of these scripts make sense in the concept of our CDK library. For example, does the `compat` script work? I would dig into this area a bit more and make sure all of these scripts serve a purpose.

Reply:
I removed the scripts that were unnecessary for now and put them on a list to possibly revisit later.